components/board/client.go

```diff
@@ -38,8 +38,6 @@ type client struct {
 
 type boardInfo struct {
 	name                  string
-	spiNames              []string
-	i2cNames              []string
```

Comment:
Surprise: these get read, but they never get written! They've been empty this entire time.